lib/rspec/rails/swagger/helpers.rb

```diff
@@ -45,7 +45,7 @@ def path template, attributes = {}, &block
             #TODO template might be a $ref
             meta = {
               swagger_object: :path_item,
-              swagger_document: attributes[:swagger_document] || RSpec.configuration.swagger_docs.keys.first,
+              swagger_document: attributes[:swagger_document] || default_document,
```

Comment:
Great work! 🥇 

Would you like to use `swagger_doc` instead of `swagger_document` to align with rswag? or add usage in readme is good thing as well 👍

Reply:
I think I'll keep the current naming just because it's currently getting past to child contexts that way and it'd be a little weird having two different names for it. Adding to the docs is a good idea though.

Reply:
Ah seeing what you're saying in #43 about https://github.com/drewish/rspec-rails-swagger/blob/5e6127c08f2a4e362204a525267e02b5ad5ebad2/spec/swagger_helper.rb#L16 
and the config using the `swagger_docs` name... it would make sense to pick one. I think it makes sense to change that and then bump the version to indicate the compatibility break.

Reply:
It's pretty great work! Thanks @drewish. 👍 👍 👍, You made my life easy. Hope it would be released soon. 🎉🎉🎉